easybuild/easyblocks/t/trinity.py

```diff
@@ -279,7 +286,16 @@ def install_step(self):
             else:
                 raise EasyBuildError("Don't know how to set TRINITY_COMPILER for %s compiler", comp_fam)
 
-            cmd = "make TRINITY_COMPILER=%s" % trinity_compiler
+            explicit_make_args = ''
+            if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                explicit_make_args = 'all plugins'
+            # Delete CXXFLAGS CPPFLAGS and CFLAGS (interfere with plugins installation)
+            if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                del os.environ['CXXFLAGS']
+                del os.environ['CPPFLAGS']
+                del os.environ['CFLAGS']
```

Comment:
The problem I bumped into in https://github.com/hpcugent/easybuild-easyconfigs/pull/2306#issuecomment-170876129 is related to this.

The reason that this is needed seems to be that `gcc` is hardcoded for the plugins, and using Intel compiler flags with GCC isn't working out well.

You're not undefining `$LIBS` here, which explains the problem I'm running into.

However, I feel this is wrong approach: we should make sure the Intel compilers are used instead where they should be?